src/services/user.service.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+let users = [];
+let increment = 1;
```

Comment:
Why do we need let in this case?
Don`t see any reason why we don't use const for this array.
Try rewriting using const. We avoid not needed assignments.

Reply:
I see this assignment. 
We can manipulate with an array. 
Any let assigned makes code more dirty readability.
Please try rewrite, also in this case you don't need ` let increment = 1;` part of code.
Thanks!